vmhost/vmhooks/generate/eiGenWriteRustWasmerMeteringHelpers.go

```diff
@@ -30,9 +27,24 @@ func WriteRustWasmerMeteringHelpers(out *eiGenWriter) {
 	var content string
 	for fileScanner.Scan() {
 		line := fileScanner.Text()
+		if strings.ToLower(line) == localCostName {
```

Comment:
This is ok for now, but moving forward, we should create 2 lists: one with wasmer2 operators, and one with whatever else we use in the executor.

Reply:
Agreed

Reply:
For now, you could extract this into a separate function which returns a boolean, deciding whether to keep the opcode or `continue` over it. Then any future changes will be isolated there.

Indeed, the `if` is a little crude, but it's completely sufficient for now.